packages/flame/lib/src/events/flame_game_mixins/multi_tap_dispatcher.dart

```diff
@@ -118,9 +125,9 @@ class MultiTapDispatcher extends Component implements MultiTapListener {
 
   //#region MultiTapListener API
 
-  /// The delay (in seconds) after which a tap is considered a long tap.
+  /// The delay (in milliseconds) after which a tap is considered a long tap.
   @override
-  double get longTapDelay => 0.300;
+  int get longTapDelay => _longTapDelay;
```

Comment:
In Flame we use seconds everywhere, so this should not be changed to milliseconds